src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -30,7 +30,7 @@ describe('Test suite for dictionary concepts components', () => {
   const props = {
     match: {
       params: {
-        conceptType: 'question',
+        conceptType: CLASS_QUESTION,
```

Comment:
Is it a class here too?

Reply:
It is a class

Reply:
Then why are you assigning it to a concept type yet when it is a class?

Reply:
I have changed it